### PR TITLE
Docs: apply sentence-style capitalization

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Docs: apply sentence-style capitalization

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/automation/trigger.markdown [hunk 1]
```diff
@@ -971,7 +971,7 @@ automation:
       event: sunset
 ```
 
-## Multiple Entity IDs for the same Trigger
+## Multiple entity IDs for the same Trigger
```
Comment (klaasnicolaas) [inline source/_docs/automation/trigger.markdown]: Trigger? or trigger?

Comment (c0ffeeca7) [inline source/_docs/automation/trigger.markdown]:
```suggestion
## Multiple entity IDs for the same trigger
```

Comment (c0ffeeca7) [reply source/_docs/automation/trigger.markdown]: Thank you for the catch!

**Review by klaasnicolaas — APPROVED** [review]
✅ Approved

**Pull request merged**